posydon/binary_evol/DT/magnetic_braking.py

```diff
@@ -0,0 +1,361 @@
+    # dOmega_mb/dt is in units of [yr^-2].
+
+    Parameters
+    ----------
+    p_pri, p_sec : two tuples
+        The properties of the primary and secondary stars, respectively. Each
+        tuple contains, in order, M_pri (the star's mass in Msun), R_pri
+        (the star's radius in Rsun), Omega_pri (the star's angular velocity in
+        1/yr), I_pri (the star's moment of inertia in Msun Rsun^2),
+        tau_conv_pri (the convective timescale of the primary),
+        and Mdot_pri (the mass loss rate of the star in Msun/yr)
+
+    Returns
+    -------
+    dOmega_mb_sec, dOmega_mb_pri : float
+        The spin angular velocity derivative for the secondary and primary
+        stars, respectively [units of yr^-2]
+
+    """
+    M_pri, R_pri, Omega_pri, I_pri, tau_conv_pri, Mdot_pri = p_pri
+    M_sec, R_sec, Omega_sec, I_sec, tau_conv_sec, Mdot_sec = p_sec
+
+    # Converting units:
+    # The constant 3.8e-30 from Rappaport+1983 has units of [cm^-2 s]
+    # which need to be converted...
+    #
+    # -3.8e-30 [cm^-2 s] * (const.rsol**2/const.secyer) -> [Rsol^-2 yr]
+    # * M [Msol]
+    # * R ** 4 [Rsol^4]
+    # * Omega ** 3 [yr^-3]
+    # / I [Msol Rsol^2 ]
+    #
+    # Thus, dOmega/dt comes out to [yr^-2]
+
+    dOmega_mb_sec = (
+        -3.8e30 * (const.rsol**2 / const.secyer)
+        * M_sec
+        * R_sec**4
+        * Omega_sec**3
+        / I_sec
+        * np.clip((1.5 - M_sec) / (1.5 - 1.3), 0, 1)
+    )
+    dOmega_mb_pri = (
+        -3.8e30 * (const.rsol**2 / const.secyer)
+        * M_pri
+        * R_pri**4
+        * Omega_pri**3
+        / I_pri
+        * np.clip((1.5 - M_pri) / (1.5 - 1.3), 0, 1)
+    )
+
+    return dOmega_mb_sec, dOmega_mb_pri
+
+
+def calculate_magnetic_braking_M15(p_pri, p_sec):
+    """Use the Matt et al. (2015) prescription.
+
+    Calculate the impact of magnetic braking on a binary using the
+    Matt et al. 2015, ApJ, 799, 23 prescription.
+
+    Parameters
+    ----------
+    p_pri, p_sec : two tuples
+        The properties of the primary and secondary stars, respectively. Each
+        tuple contains, in order, M_pri (the star's mass in Msun), R_pri
+        (the star's radius in Rsun), Omega_pri (the star's angular velocity in
+        1/yr), I_pri (the star's moment of inertia in Msun Rsun^2),
+        tau_conv_pri (the convective timescale of the primary),
+        and Mdot_pri (the mass loss rate of the star in Msun/yr)
+
+    Returns
+    -------
+    dOmega_mb_sec, dOmega_mb_pri : float
+        The spin angular velocity derivative for the secondary and primary
+        stars, respectively [units of yr^-2]
+
+    """
+    M_pri, R_pri, Omega_pri, I_pri, tau_conv_pri, Mdot_pri = p_pri
+    M_sec, R_sec, Omega_sec, I_sec, tau_conv_sec, Mdot_sec = p_sec
+
+    # Torque prescription from Matt et al. 2015, ApJ, 799, L23
+    # Constants:
+    # [erg] or [g cm^2 s^-2] -> [Msol Rsol^2 yr^-2]
+    K = 1.4e30 * const.secyer**2 / (const.msol * const.rsol**2)
```

Comment:
```suggestion
    K = -1.4e30 * const.secyer**2 / (const.msol * const.rsol**2)
```

Right now the torque in M15 would evaluate to positive. The constant K should have a negative sign